xdsl/dialects/builtin.py

```diff
@@ -782,6 +782,27 @@ def unpack(
 
 
 BoolAttr: TypeAlias = IntegerAttr[Annotated[IntegerType, IntegerType(1)]]
+TRUE_ATTR = BoolAttr(True, i1)
```

Comment:
I am in general against the addition of more helper methods that do the same things in questionably more ergonomic syntax. There is already `BoolAttr.from_bool()` and I feel we should only keep one or the other.

Reply:
I had a feeling we had such a helper but forgot where it was.